.travis.yml

```diff
@@ -18,7 +18,6 @@ addons:
 jdk:
   - oraclejdk11
   - oraclejdk8
-  - openjdk7
```

Comment:
Do we want to try *some* OpenJDK?

Reply:
Could probably do `openjdk11` if that was useful.

Reply:
I could see the benefit of having one if not two openjdks as well. For reference:
![screen shot 2018-12-10 at 13 25 57](https://user-images.githubusercontent.com/88113/49732491-2a911f00-fc7f-11e8-9e88-69000f79f6e2.png)

Reply:
Given the JDK ecosystem, within the scope of this PR I would be inclined to go for a hybrid solution of type:

```
oraclejdk11 / maven
openjdk11 / maven
oraclejdk8 / maven
openjdk8 / ant
```

as a compromise between build systems/JDKs coverage and testing speed. Thoughts?

Reply:
Have you fully read up on the Oracle [licensing changes](https://blogs.oracle.com/java-platform-group/oracle-jdk-releases-for-java-11-and-later) for JDK11?  [One writeup](https://blog.joda.org/2018/09/do-not-fall-into-oracles-java-11-trap.html).

Since Oracle JDK can't be used for any production purpose without paying Oracle or risk being sued, the vast majority of our users, on all platforms, are going to be better off using OpenJDK from this point onward.  It would make sense to prioritise testing on OpenJDK 11 in consequence.